Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -7,9 +7,9 @@
 
 public class ContestValidationService : IContestValidationService
 {
-    public ValidationResult GetValidationResult((Contest?, int?, string, bool, bool) item)
+    public ValidationResult GetValidationResult((Contest?, int?, string, bool, bool, DateTime) item)
```

Comment:
The date should not a be an external dependency. Better inject IDatesService in the constructor and get the date internally. In order for the validation service to do it's job, it should use only the current date, not an arbitrary date, so there is no need to transfer the responsibility to the caller, to provide the correct date.